libs/ledger/tests/unit/stake_manager_tests.cpp

```diff
@@ -42,6 +42,8 @@ using StakeManagerPtr = std::unique_ptr<StakeManager>;
 using EntropyPtr      = std::unique_ptr<NaiveEntropyGenerator>;
 using RoundStats      = std::unordered_map<Identity, std::size_t>;
 
+constexpr uint64_t MAX_COMMITTEE_SIZE = 1;
```

Comment:
That seems rather small.

Reply:
It's just carried over from the old stake manager tests which ed wrote